stock_available/views/product_product_view.xml

```diff
@@ -60,14 +60,6 @@
         <field name="model">product.product</field>
         <field name="inherit_id" ref="stock.view_stock_product_tree" />
         <field name="arch" type="xml">
-            <tree position="attributes">
-                <attribute
-                    name="decoration-danger"
-                >virtual_available&lt;0 or immediately_usable_qty&lt;0</attribute>
-                <attribute
-                    name="decoration-info"
-                >virtual_available&gt;=0 or immediately_usable_qty&gt;0</attribute>
-            </tree>
             <field name="virtual_available" position="after">
                 <field name="immediately_usable_qty" />
```

Comment:
Instead of just removing it, I would add the decoration on the field itself and add the group, exactly as Odoo is doing in the code you pointed: https://github.com/odoo/odoo/blob/16.0/addons/stock/views/product_views.xml#L59